dao-voting/tests/utils/mod.rs

```diff
@@ -0,0 +1,195 @@
+use fuels::{prelude::*, tx::ContractId};
+
+// Load abi from json
+abigen!(DaoVoting, "out/debug/dao-voting-abi.json");
+abigen!(
+    GovToken,
+    "tests/artifacts/gov_token/out/debug/gov_token-abi.json"
+);
+
+pub struct Metadata {
+    pub dao_voting: DaoVoting,
+    pub gov_token: Option<GovToken>,
+    pub wallet: LocalWallet,
+}
+
+pub mod abi_calls {
+    use fuels::contract::contract::CallResponse;
```

Comment:
Since you're using `use super::*` below we should have access to everything so this line should not be needed.